<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="robot_name" default="robot"/>

  <group ns="$(arg robot_name)">

      <node name="ltl_planner" pkg="rqt_simulation" type="ltl_planner.py" output="screen"/>

  </group>

</launch>
